cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -854,3 +854,84 @@ def _revert():
     view = navigate_to(ansible_service, "Details")
     assert view.provisioning.credentials.get_text_of("Vault") == vault_creds.name
     assert view.provisioning.results.get_text_of("Status") == "successful"
+
+
+@pytest.mark.meta(automates=[BZ(1444092)])
+@pytest.mark.parametrize(
+    "credential", CREDENTIALS, ids=[cred[0] for cred in CREDENTIALS]
+)
+@pytest.mark.provider(
+    [RHEVMProvider, EC2Provider, VMwareProvider, AzureProvider], selector=ONE_PER_TYPE
+)
+@pytest.mark.uncollectif(
+    lambda credential, provider: not (
+        (credential[0] == "Amazon" and provider.one_of(EC2Provider))
+        or (credential[0] == "VMware" and provider.one_of(VMwareProvider))
+        or (
+            credential[0] == "Red Hat Virtualization" and provider.one_of(RHEVMProvider)
+        )
+        or (credential[0] == "Azure" and provider.one_of(AzureProvider))
+    )
+)
+@pytest.mark.tier(3)
+def test_ansible_service_cloud_credentials(
+    appliance,
+    request,
+    ansible_catalog_item,
+    ansible_service,
+    ansible_service_catalog,
+    credential,
+    provider_credentials,
+):
+    """
+        When the service is viewed in my services it should also show that the cloud credentials
+    were attached to the service.
+
+    Bugzilla:
+        1444092
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: medium
+        initialEstimate: 1/4h
+        tags: ansible_embed
+    """
+    playbook = credential[2]
+    with update(ansible_catalog_item):
+        ansible_catalog_item.provisioning = {
+            "playbook": playbook,
+            "cloud_type": provider_credentials.credential_type,
+            "cloud_credential": provider_credentials.name,
+        }
+
+    @request.addfinalizer
+    def _revert():
+        with update(ansible_catalog_item):
+            ansible_catalog_item.provisioning = {
+                "playbook": "dump_all_variables.yml",
+                "cloud_type": "<Choose>",
+            }
+
+        service = MyService(appliance, ansible_catalog_item.name)
+        if service_request.exists():
```

Comment:
Comment for reviewers: This tear down is already written in `funcscope` fixture, in this PR #9194